doc/source/launcher.rst

```diff
@@ -1,101 +0,0 @@
-.. _launcher:
-
-Launcher
-========
-
-The Kivy Launcher is an Android application that can run any Kivy app
-stored in the `kivy` folder on the SD Card. You can download the latest stable
-version for your android device from the
-`Play Store <https://play.google.com/store/apps/details?id=org.kivy.pygame>`_.
```

Comment:
FYI:
There's a still-in-WIP reboot of the old Kivy launcher, hosted here: https://github.com/kivy/kivy-launcher
 
 But, I agree that launcher docs in `python-for-android` are old and unmaintained.
 
 Also, if the new launcher will do a comeback into the Kivy ecosystem, I feel we should advertise it on kivy.org website, and not into `python-for-android` docs. (as `kivy-launcher` will likely need to have its own docs)

Reply:
Yes, all agreed. The WIP reboot isn't mature enough to be referenced here, and seems to have stalled.